db/migrate/20221202120901_remove_unused_authentication_token.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveUnusedAuthenticationToken < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :spree_users, :authentication_token, :string
```

Comment:
Can we just remove the column for new stores in https://github.com/solidusio/solidus_auth_devise/blob/main/db/migrate/20101026184950_rename_columns_for_devise.rb?